rx-extensions/src/main/java/com/appunite/rx/util/LogTransformer.java

```diff
@@ -34,36 +31,43 @@
 
         Logger DEFAULT = new Logger() {
             @Override
-            public void logSubscribe(@NonNull String tag, @Nonnull String observableTag) {
+            public void logOnSubscribe(@Nonnull final String tag, @Nonnull final String observableTag) {
                 final java.util.logging.Logger logger = java.util.logging.Logger.getLogger(tag);
-                logger.log(Level.INFO, observableTag + " -> onSubscribe");
+                if (logger.isLoggable(Level.INFO)) {
```

Comment:
logger internally checks log level, no reason to duplicate behavior here

this check makes sense only for error (Throwable) and onNext (Object) so we can avoid expensive .toString call on objects